tests/resources/Engine.py

```diff
@@ -3757,6 +3794,33 @@ def ctn_get_host_info_grpc(id:int):
                     logger.console(f"gRPC server not ready {e}")
     return {}
 
+def ctn_get_service_info_grpc(id_h:int,id_s:int):
+    """
+    Retrieve service information via a gRPC call.
+
+    Args:
+        id_h: The identifier of the host to retrieve.
+        id_s: The identifier of the service to retrieve.
+
+    Returns:
+        A dictionary containing the service informations, if successfully retrieved.
+    """
+    if id_h is not None and id_s is not None:
```

Comment:
As id_h and id_s are declared as int, is this test relevant?

Reply:
will fix, the test isn't necessary.